client/my-sites/plugins/sidebar/style.scss

```diff
@@ -1,6 +1,23 @@
 .global-sidebar .sidebar--plugins {
 	.sidebar__back-link {
-		margin-bottom: 1rem;
+		margin-bottom: 0;
+		a {
+			padding-inline: 0;
+		}
+		.sidebar__site-title {
+			font-size: 18px; /* stylelint-disable-line declaration-property-unit-allowed-list */
+			font-weight: 500;
+		}
+	}
+
+	.sidebar__sub-heading {
+		font-size: 13px; /* stylelint-disable-line declaration-property-unit-allowed-list */
+		padding-left: 20px;
```

Comment:
It should be padding-inline-start to avoid RTL issues. But it seems to be better to add the padding to both sides.

```css
padding: 0 20px;
```

Reply:
Fixed [here](https://github.com/Automattic/wp-calypso/pull/91965/commits/b13569944dbe092ecc8be07541cc78d63f006453)